### PR TITLE
Catalog.

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_catalog/)
- [TEST REPORT LINK](https://OleksandrFEdev.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/block/card.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  line-height: 14px;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
```
Comment (loralevitska) [inline src/styles/block/card.scss:13]: It is a good practice to make all colors variable. Fix all cases

File: src/styles/block/card.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
```
Comment (loralevitska) [inline src/styles/block/card.scss:5]: Not part of card. Move it to `main.scss`

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job! There are things that should be fixed:
Please fix this point 
image
image

File: src/styles/block/catalog.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  &__card {
+    margin-bottom: 46px;
+    margin-right: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/styles/block/catalog.scss:22]: and it should be 4 cards in a row

File: src/styles/block/card.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    &-number {
+      font-size: 16px;
+      line-height: 19px;
+      font-weight: 700;
+      color: $color-main-accent;
+    }
+  }
+
+  &__button {
+    width: 166px;
```
Comment (Viktor-Kost) [inline src/styles/block/card.scss:68]: no needs to fix width for your button as you have paddings and fixed width for your card
```suggestion
    width: 100%;
```

File: src/styles/block/stars.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.stars__star {
+  height: 12px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```
Comment (Viktor-Kost) [inline src/styles/block/stars.scss:7]: 1) apply background `shorthand`
2) probably you need to use background-position: center;

File: src/styles/block/stars.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding: 0 2px 2px;
```
Comment (Viktor-Kost) [inline src/styles/block/stars.scss:14]: it's not a good practice to center like this
```suggestion
```

File: src/index.html [hunk 7]
```diff
@@ -4,9 +4,474 @@
...
+          alt="product-img"
+        >
+
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock
+          (MNED2UA/A)
+        </div>
+
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
```
Comment (Viktor-Kost) [inline src/index.html:121]: use semantic tags for text elements. Check other cases
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>

        <p class="card__product-code">
          Product code: 195434
        </p>
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!)

**Pull request closed without merging**